packages/discord-request/src/manager.ts

```diff
@@ -175,6 +175,7 @@ export class Manager {
 			return;
 		}
 
+		// @ts-expect-error Not sure why this started breaking
```

Comment:
Not sure what happened, but these types fell back to node. I wonder if it's the undici inclusion?